canonical-web-and-design/ubuntu.com
# Add release names on the chart

## Done

- Added release names on `/about/release-cycle`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/about/release-cycle
- Be sure to test on mobile, tablet and desktop screen sizes
- [Demo](https://ubuntu-com-10919.demos.haus/about/release-cycle)


## Issue / Card

Fixes #10850 

## Screenshots

![image](https://user-images.githubusercontent.com/57550290/142885822-e9ee5a9d-1a1a-4b33-8d04-f6a718c1759c.png)

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
